commit 7dcca30a32aadb0520417521b0c44f42d09fe05c
Author: Linus Torvalds <torvalds@woody.linux-foundation.org>
Date:   Sun Jul 8 16:32:17 2007 -0700

    Linux 2.6.22
    
    Woo-hoo. I'm sure somebody will report a "this doesn't compile, and
    I have a new root exploit" five minutes after release, but it still
    feels good ;)
    
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>

diff --git a/Makefile b/Makefile
index c6f7b71eae81..de4f8f7d396c 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 2
 PATCHLEVEL = 6
 SUBLEVEL = 22
-EXTRAVERSION = -rc7
+EXTRAVERSION =
 NAME = Holy Dancing Manatees, Batman!
 
 # *DOCUMENTATION*